scripts/st2bootstrap-deb.sh

```diff
@@ -96,6 +96,30 @@ setup_args() {
   fi
 }
 
+check_st2_host() {
+  # Check that the following TCP ports are available.
+  # Abort the installation early if the required ports are being used by an existing process.
+
+  # nginx (80, 443), mongodb (27017), rabbitmq (4369, 5672, 25672), and st2 (9100-9102).
```

Comment:
Agreed. It would be helpful to describe specifically what happened and how to fix it. Thanks.

As is, the lsof command outputs something like this:

```
gunicorn_  1007      st2    5u  IPv4  23210      0t0  TCP localhost:9102 (LISTEN)
gunicorn_  1025      st2    5u  IPv4  23208      0t0  TCP localhost:9101 (LISTEN)
epmd       1212 rabbitmq    3u  IPv4  21531      0t0  TCP *:4369 (LISTEN)
epmd       1212 rabbitmq    4u  IPv6  21532      0t0  TCP *:4369 (LISTEN)
beam.smp   1411 rabbitmq   12u  IPv4  22126      0t0  TCP *:25672 (LISTEN)
beam.smp   1411 rabbitmq   23u  IPv6  23431      0t0  TCP *:5672 (LISTEN)
gunicorn_  9299      st2    5u  IPv4  79283      0t0  TCP localhost:9100 (LISTEN)
gunicorn_ 24704      st2    5u  IPv4  23208      0t0  TCP localhost:9101 (LISTEN)
gunicorn_ 24707      st2    5u  IPv4  23210      0t0  TCP localhost:9102 (LISTEN)
gunicorn_ 24825      st2    5u  IPv4  79283      0t0  TCP localhost:9100 (LISTEN)
```

Is it enough to follow that output with:

```
Not all required TCP ports are available. ST2 will fail to start.
Please, stop any services operating on the above LISTENING ports.
```